src/Sylius/Bundle/CurrencyBundle/spec/Form/Type/CurrencyTypeSpec.php

```diff
@@ -46,6 +46,16 @@ function it_should_build_form_with_proper_fields(FormBuilder $builder)
         ;
 
         $builder
+          ->add('enabled', 'checkbox', Argument::any())
+          ->willReturn($builder)
```

Comment:
One last update. This spec, will pass, even if we don't call this method. It should have `->shouldBeCalled()` call also. I know, that probably anyone doesn't have it, but this is an old spec and it is written bad.